CI: Remove sxstrace test from CI

imphil opened the pull request:
Make it possible to run the sxstest through nox.

Remove it from CI, since it doesn't seem to provide enough value to run
repeatedly.


<!--

Thanks for improving cocotb! Here are some points to make this as smooth as possible.
Not all of them may be applicable.

Most important: please explain *why* you are proposing this change.

* Make sure you have read https://github.com/cocotb/cocotb/blob/master/CONTRIBUTING.md
* Extend or add a test under `tests/test_cases/`.
* Add documentation under `documentation/source/`,
  docstrings in Python code, or Doxygen markup in C/C++ code.
  Use ``versionadded``/``versionchanged``/``deprecated``.
* Add a newsfragment - see `documentation/source/newsfragments/README.rst`.
* Use `closes #XXXX` to auto-close the issue that this PR fixes (if such).

-->

imphil commented:
@ktbarrett I'm very unsure about the value of this test (and the script in general). Please let me know how to proceed. (We can keep it in CI, but it would be additional work, that I thought I could avoid.)

ktbarrett reviewed (approved):
It's useful for debugging sxs configuration issues. It never belonged in CI. Ideally I'd like to restructure the C++ libraries so we don't need that stuff, since I don't think any of the maintainers understand it.

ktbarrett commented:
Must fix formatting first ofc.

imphil commented:
> Ideally I'd like to restructure the C++ libraries so we don't need that stuff, since I don't think any of the maintainers understand it.

I repeatedly understood it, and forgot the details again roughly half a day later :-)

imphil commented:
@ktbarrett would you mind having a quick look again, since I now removed the nox part, and just added the command as comment to the powershell script. (The nox session depended on functionality in the wheelbuild PR,  and I don't think it's super valuable to have this invocation in nox anyways.)
